data/preset/buffet.yaml

```diff
@@ -118,7 +121,9 @@ defines:
                             - 0
                             - 1
                     selection:
-                      type: Selection.AllHighlights
+                      type: Selection.FeaturedFirst
```

Comment:
when testing travel app, I noticed something weird, when you open the app.. you see the content group containing this selection loading cards one by one quite slowly.